### PR TITLE
Added license and overview page

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

**Comment by fhennig** [comment]
@stefanigel could you review this?

This is the product information page: https://deploy-preview-439--stackable-docs.netlify.app/home/nightly/product-information

and this is the license page: https://deploy-preview-439--stackable-docs.netlify.app/home/nightly/licenses

Additionally every demo now has system requirements and an installation section: https://deploy-preview-439--stackable-docs.netlify.app/stackablectl/stable/demos/

And stackablectl also has system requirements: https://deploy-preview-439--stackable-docs.netlify.app/stackablectl/stable/installation#system-requirements

The only thing missing from the ticket is the SBOMs, but they are not ready yet.

Link to ticket: https://github.com/stackabletech/internal-issues/issues/43

File: modules/ROOT/pages/product-information.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
+= Product information
+
+This page contains contrete specifications about the Stackable Data Platform (SDP) as a product, which components are included, how they are supplied and which external dependencies exist that you as a customer need to take care of.
+
+[#platform-components]
+== Platform components
+
+The Stackable Platform is made up of multiple components. Operators for data products, Operators for additional functionality to facilitate easy integration between the different products. Auxilary software to interact with the platform, as well as software to ease the deployment of platform components.
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:8]: and the data products themselves

File: modules/ROOT/pages/product-information.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
+= Product information
+
+This page contains contrete specifications about the Stackable Data Platform (SDP) as a product, which components are included, how they are supplied and which external dependencies exist that you as a customer need to take care of.
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:3]: concrete

File: modules/ROOT/pages/product-information.adoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+[#platform-components]
+== Platform components
+
+The Stackable Platform is made up of multiple components. Operators for data products, Operators for additional functionality to facilitate easy integration between the different products. Auxilary software to interact with the platform, as well as software to ease the deployment of platform components.
+
+=== Kubernetes Operators
+
+The main components of the SDP are Kubernetes Operators. Each Operator is supplied in a Docker <<containers, container>> image. For every Operator there is also a Helm Chart to facilitate installation via Helm. Operators for Products supported by the Platform deploy these products as <<containers, containers>> in Kubernetes.
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:12]: The main components of the SDP are a set of open source data products. The SDP control plane consists of corresponding  Kubernetes Operators.

Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:12]:
```suggestion
The main components of the SDP are a set of open source data products. The SDP control plane consists of corresponding Kubernetes Operators. Each Operator is supplied in a Docker <<containers, container>> image. For every Operator there is also a Helm Chart to facilitate installation via Helm. Operators for Products supported by the Platform deploy these products as <<containers, containers>> in Kubernetes.
```

File: modules/ROOT/pages/product-information.adoc (hunk 2 above)
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:3]:
```suggestion
This page contains concrete specifications about the Stackable Data Platform (SDP) as a product, which components are included, how they are supplied and which external dependencies exist that you as a customer need to take care of.
```

File: modules/ROOT/pages/product-information.adoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+* xref:hbase:index.adoc[Apache HBase] (including xref:hbase:usage-guide/phoenix.adoc[Apache Phoenix])
+* xref:hive:index.adoc[Apache Hive Metastore]
+* xref:hdfs:index.adoc[Apache Hadoop HDFS]
+* xref:kafka:index.adoc[Apache Kafka]
+* xref:nifi:index.adoc[Apache Nifi]
+* xref:spark-k8s:index.adoc[Apache Spark] (including xref:spark-k8s:usage-guide/history-server.adoc[Spark History Server])
+* xref:superset:index.adoc[Apache Superset]
+* xref:trino:index.adoc[Trino]
+* xref:zookeeper:index.adoc[Apache Zookeeper]
+* xref:opa:index.adoc[Open Policy Agent]
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:27]: I do not see OPA as a supported data product, but as a "helper" to enable Authorization. I would like to take it from the list of products.

Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:27]:
```suggestion
```

File: modules/ROOT/pages/product-information.adoc [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+The product also includes the xref:commons-operator:index.adoc[Commons], xref:secret-operator:index.adoc[Secret] and xref:listener-operator:index.adoc[Listener] Operators.
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:29]: add opa operator here instead

Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:29]:
```suggestion
The product also includes operators that provide additional functionality for managing and control SDP, i.e.the xref:commons-operator:index.adoc[Commons], xref:secret-operator:index.adoc[Secret] and xref:listener-operator:index.adoc[Listener] Operators.
```

File: modules/ROOT/pages/product-information.adoc [hunk 6]
```diff
@@ -0,0 +1,104 @@
+= Product information
+
+This page contains contrete specifications about the Stackable Data Platform (SDP) as a product, which components are included, how they are supplied and which external dependencies exist that you as a customer need to take care of.
+
+[#platform-components]
+== Platform components
+
+The Stackable Platform is made up of multiple components. Operators for data products, Operators for additional functionality to facilitate easy integration between the different products. Auxilary software to interact with the platform, as well as software to ease the deployment of platform components.
+
+=== Kubernetes Operators
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:10]:
```suggestion
=== Data Products and Kubernetes Operators
```

File: modules/ROOT/pages/product-information.adoc [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+xref:stackablectl::index.adoc[stackablectl] is a commandline utility that makes it easier to install and interact with stackable components.
+
+== Supported installation methods
+
+[#containers]
+=== Operators and products
+
+All operators are supplied in container images. The products are also deployed in container images.
+The docker images can be found here: https://repo.stackable.tech/#browse/browse:docker
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:45]:
```suggestion
The docker images are available for download here: https://repo.stackable.tech/#browse/browse:docker
```

File: modules/ROOT/pages/product-information.adoc [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+Hardware requirements and supported operating systems can be found in the xref:stackablectl::installation.adoc#system-requirements[system requirements section] of the stackablectl documentation.
+
+=== Stackable Data Platform Open Source Products
+
+The system requirements of each open source product depend on your specific use case. In the xref:stackablectl::demos/index.adoc[demos] you can find example use cases; every demo also has a _System requirements_ section which can provide a guideline for system sizing. Further example sizings can be found at https://ci.stackable.tech/job/<product>-operator-it-summary/ (where `<product>` is i.e. _druid_, _hbase_, etc.) in the `cluster-info.txt` file. These are the cluster sizes used for integration tests, and can be regarded as working cluster configurations.
+
+== Prerequisites and required external components
+
+Required external components are components that are required for the platform or parts of it to operate, but are not part of the Stackable Data Platform. Note that Stackable does not distribute or offer support for the external components.
```
Comment (stefanigel) [inline modules/ROOT/pages/product-information.adoc:79]:
```suggestion
Required external components are components that are required for the platform or parts of it to operate, but are not part of the Stackable Data Platform. Note that Stackable does neither distribute nor offer support for the external components.
```

File: modules/ROOT/partials/supported-kubernetes-distributions.adoc [hunk 9]
```diff
@@ -0,0 +1,7 @@
+* Amazon Elastic Kubernetes Service
```
Comment (stefanigel) [inline modules/ROOT/partials/supported-kubernetes-distributions.adoc:1]: Nice to have: could we add URLs? but maybe too hard to maintain ...

File: modules/ROOT/partials/supported-kubernetes-distributions.adoc [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+* Microsoft Azure Kubernetes Service
+* Google Kubernetes Engine
+* IONOS Managed Kubernetes
+* SUSE Rancher
+* SUSE K3S
+* Red Hat OpenShift
```
Comment (stefanigel) [inline modules/ROOT/partials/supported-kubernetes-distributions.adoc:7]: OpenShift has its own Versioning schema: 4.10, 4.11, 4.12 ...

File: modules/ROOT/pages/kubernetes.adoc [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+The Stackable Data Platform requires a Kubernetes cluster to be present, where you can install things into. How to set up Kubernetes as well as a cluster depends on the distribution you chose. 
+
+The following distributions are supported for a production setup of the Stackable Data Platform:
+
+include::partial$supported-kubernetes-distributions.adoc[]
+
+In this version of the SDP, the following Kubernetes versions are supported:
+
+include::partial$supported-kubernetes-versions.adoc[]
```
Comment (stefanigel) [inline modules/ROOT/pages/kubernetes.adoc:16]: Is this the actual list of versions in this release? --> refer to releases?
And: OpenShift has its own versioning

**Review by stefanigel — CHANGES_REQUESTED** [review]
The most important issues:
- Also the data products themselves are part of SDP and a support subscription
- Supported K8s versions seems to be outdated?

File: modules/ROOT/pages/product-information.adoc (hunk 4 above)
Comment (fhennig) [reply modules/ROOT/pages/product-information.adoc:27]: Ok I'll do it. I just grouped "product" vs "ours"

**Review by stefanigel — APPROVED** [review]

**Pull request merged**